app/views/products/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<% presenter = ProductPresenter.new(@product) %>
+
+<div class="container my-3">
+  <div class="d-flex justify-content-around align-items-center py-4">
+    <h2 class="text-center"><%= @product.title %></h2>
+    <h5><%= @product.category if @product.category.present? %></h5>
+  </div>
+
+  <div class="d-lg-flex">
+    <div class="col-lg-4">
+      <%= pass_image(@product) %>
+    </div>
+    <div class="col-lg-8 pt-2 pt-md-0">
+      <p class="pl-3">Description: <%= @product.description if @product.description.present? %></p>
+      <p class="pl-3">Brand: <%= @product.brand %></p>
+      <p class="pl-3">Manufacturer: <%= @product.manufacturer %></p>
+      <p class="pl-3 bg-<%= presenter.new_product_marker %>">Year: <%= @product.year if @product.year.present? %></p>
```

Comment:
Please, add the condition of @product.year to 'Year:' too
Because, in this case, if @product.year == nil Year will display